charts/opensearch/values.yaml

```diff
@@ -259,6 +259,7 @@ enableServiceLinks: true
 protocol: https
 httpPort: 9200
 transportPort: 9300
+metricsPort: 9600
```

Comment:
Can you also update values for one of the CI chart values?

Reply:
Done.